Firestore/Source/Remote/FSTRemoteStore.mm

```diff
@@ -497,13 +495,13 @@ - (void)addBatchToWritePipeline:(FSTMutationBatch *)batch {
 
   if ([self shouldStartWriteStream]) {
     [self startWriteStream];
```

Comment:
This was reworked to no longer start the write stream (https://github.com/firebase/firebase-android-sdk/blob/98326c431a189bc84829ad0c7df0089202f57990/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/RemoteStore.java#L568) and the method comment was tweaked as such as well.